src/scripts/markdownChecker.ts

```diff
@@ -39,11 +39,18 @@ const CASE_SENSITVE_SPELLING_MISTAKES = ["Thereum", "Metamask", "Github"]
 // Regex for explicit lang path (e.g. /en/) && for glossary links (trailing slash breaks links e.g. /glossary/#pos/ doesn't work)
 // We should have case sensitive spelling mistakes && check they are not in links.
 
-const langsArray = fs.readdirSync(PATH_TO_INTL_MARKDOWN)
+interface Languages {
+  lang?: Array<string>
```

Comment:
```suggestion
  lang?: Array<Lang>
```

I was testing this out, and I think you can use this if you import the Lang type

```
import { Lang } from "../utils/languages"
```
Can use this if its any help.

Reply:
Thanks @corwintines :rocket: